<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Results (powered by Planemo)"
           tests="50"
           errors="5"
           failures="6"
           skip="0">
    
    <testcase classname="bam_to_sam (Test #1)" name="0" time="36.28526592254639">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bam_to_sam (Test #2)" name="1" time="33.24872279167175">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bam_to_sam (Test #3)" name="2" time="32.94096326828003">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #1)" name="0" time="38.01780128479004">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #2)" name="1" time="34.64238882064819">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #3)" name="2" time="30.879361152648926">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #4)" name="3" time="32.26427173614502">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #5)" name="4" time="19.216522216796875">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1434, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 747, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 839, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1434, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 747, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 839, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 4, "time_seconds": 19.216522216796875, "tool_id": "bcftools_merge", "tool_version": "1.15.1+galaxy4"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #6)" name="5" time="33.34853196144104">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [W::bcf_sr_add_reader] Using multiple unindexed files may produce errors, make sure chromosomes are in the same order!
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_merge (Test #7)" name="6" time="31.909281015396118">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_plugin_missing2ref (Test #1)" name="0" time="40.18685483932495">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Filled 44 REF alleles
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="column_order_header_sort (Test #1)" name="0" time="16.25844669342041">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1434, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 747, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 839, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1434, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 747, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 839, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 0, "time_seconds": 16.25844669342041, "tool_id": "column_order_header_sort", "tool_version": "0.0.1"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="deeptools_plot_pca (Test #1)" name="0" time="50.026909589767456">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="deeptools_plot_pca (Test #2)" name="1" time="36.976563453674316">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #1)" name="0" time="74.89428949356079">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Detecting adapter sequence for read1...No adapter detected for read1Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Filtering result:reads passed filter: 2reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 0bases trimmed due to adapters: 0Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger fastp v0.24.0, time used: 0 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #2)" name="1" time="13.084743976593018">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1434, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 747, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 839, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1434, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 747, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 839, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 1, "time_seconds": 13.084743976593018, "tool_id": "fastp", "tool_version": "0.24.0+galaxy3"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="fastp (Test #11)" name="10" time="32.90268301963806">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Detecting adapter sequence for read1...No adapter detected for read1Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Filtering result:reads passed filter: 2reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 0bases trimmed due to adapters: 0reads with polyX in 3' end: 0bases trimmed in polyX tail: 0Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq_gz.fastqsanger.gz -i R1_fq_gz.fastqsanger.gz -o first.fastqsanger.gz -x --poly_x_min_len 10 fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #12)" name="11" time="35.2316620349884">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            WARNNIG: different read numbers of the 0 packRead1 pack size: 99Read2 pack size: 100ERROR: input files don't contain identical amount of reads
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #13)" name="12" time="36.95570158958435">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 99total bases: 24849Q20 bases: 23268(93.6376%)Q30 bases: 22391(90.1083%)Read2 before filtering:total reads: 99total bases: 24849Q20 bases: 22676(91.2552%)Q30 bases: 21532(86.6514%)Merged and filtered:total reads: 97total bases: 25703Q20 bases: 24459(95.1601%)Q30 bases: 23743(92.3744%)Filtering result:reads passed filter: 196reads failed due to low quality: 2reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 68bases trimmed due to adapters: 2824reads corrected by overlap analysis: 55bases corrected by overlap analysis: 101Duplication rate: 0%Insert size peak (evaluated by paired-end reads): 308Read pairs merged: 97% of original read pairs: 97.9798%% in reads after filtering: 100%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for bwa-mem-fastq1_fq.fastqsanger -i bwa-mem-fastq1_fq.fastqsanger -I bwa-mem-fastq2_fq_R2.fastqsanger --merge --merged_out /galaxy/server/database/objects/9/a/2/dataset_9a21da6e-2d72-45b8-b4d1-55cb993860d3.dat --out1 /galaxy/server/database/objects/8/1/4/dataset_814819f7-19ef-4d9d-98d8-c6eeb5fd5f5d.dat --out2 /galaxy/server/database/objects/e/5/8/dataset_e586c325-c557-4c3e-b2ab-9e6eff00d49b.dat --unpaired1 /galaxy/server/database/objects/d/3/c/dataset_d3c78cab-4d81-4a76-93ee-b0316b6344f9.dat --unpaired2 /galaxy/server/database/objects/7/5/c/dataset_75cb2c25-048d-40e7-83f7-2d4fc68abfbd.dat fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #14)" name="13" time="34.38800311088562">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 99total bases: 24849Q20 bases: 23268(93.6376%)Q30 bases: 22391(90.1083%)Read2 before filtering:total reads: 99total bases: 24849Q20 bases: 22676(91.2552%)Q30 bases: 21532(86.6514%)Merged and filtered:total reads: 100total bases: 26456Q20 bases: 25141(95.0295%)Q30 bases: 24378(92.1454%)Filtering result:reads passed filter: 197reads failed due to low quality: 1reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 68bases trimmed due to adapters: 2824reads corrected by overlap analysis: 55bases corrected by overlap analysis: 101Duplication rate: 0%Insert size peak (evaluated by paired-end reads): 308Read pairs merged: 97% of original read pairs: 97.9798%% in reads after filtering: 97%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for Unnamed Collection.fastqsanger -i Unnamed Collection.fastqsanger -I Unnamed Collection_R2.fastqsanger --merge --merged_out /galaxy/server/database/objects/d/7/9/dataset_d79736b9-a3ab-40e9-8069-3f042e8ab424.dat --include_unmerged fastp v0.24.0, time used: 0 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #15)" name="14" time="36.81739091873169">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 99total bases: 24849Q20 bases: 22676(91.2552%)Q30 bases: 21532(86.6514%)Read2 before filtering:total reads: 99total bases: 24849Q20 bases: 23268(93.6376%)Q30 bases: 22391(90.1083%)Read1 after filtering:total reads: 97total bases: 22942Q20 bases: 21535(93.8671%)Q30 bases: 20672(90.1055%)Read2 after filtering:total reads: 97total bases: 22942Q20 bases: 21680(94.4992%)Q30 bases: 20981(91.4524%)Filtering result:reads passed filter: 194reads failed due to low quality: 2reads failed due to too many N: 2reads failed due to too short: 0reads with adapter trimmed: 68bases trimmed due to adapters: 2824Duplication rate: 0%Insert size peak (evaluated by paired-end reads): 308JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for Unnamed Collection.fastqsanger.gz -i Unnamed Collection.fastqsanger.gz -I Unnamed Collection_R2.fastqsanger.gz -o first.fastqsanger.gz -O second.fastqsanger.gz fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #3)" name="2" time="30.971394062042236">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 1total bases: 151Q20 bases: 114(75.4967%)Q30 bases: 98(64.9007%)Filtering result:reads passed filter: 1reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 1reads with adapter trimmed: 1bases trimmed due to adapters: 139Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger --adapter_sequence ATCG fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #4)" name="3" time="26.559338092803955">
        
            
                <error type="error" message="Tool exit code: None"><![CDATA[
                    { "inputs": {  "output_options|report_json": false,  "read_mod_options|umi_processing|umi": true,  "read_mod_options|umi_processing|umi_len": "8",  "read_mod_options|umi_processing|umi_loc": "read1",  "single_paired|in1": {   "id": "06e5c12644eb3d77",   "src": "hda"  },  "single_paired|single_paired_selector": "single" }, "job": {  "command_line": "ln -sf \u0027/galaxy/server/database/objects/d/1/8/dataset_d188080c-bfad-4375-aa44-84f9973628c8.dat\u0027 \u0027R1_fq.fastqsanger\u0027 \u0026\u0026    fastp  --thread ${GALAXY_SLOTS:-1} --report_title \u0027fastp report for R1_fq.fastqsanger\u0027  -i \u0027R1_fq.fastqsanger\u0027   -o first.fastqsanger                                         -U --umi_loc \u0027read1\u0027 --umi_len 8       \u0026\u0026 mv first.fastqsanger \u0027/galaxy/server/database/objects/b/5/f/dataset_b5fa9236-471f-4ae7-8691-217b9a534eef.dat\u0027",  "command_version": null,  "copied_from_job_id": null,  "create_time": "2025-06-20T01:02:18.109407",  "dependencies": [],  "exit_code": null,  "external_id": "gxy-2cdr6",  "galaxy_version": "24.1",  "handler": null,  "history_id": "62db28f22a512698",  "id": "3087c3af202e90eb",  "inputs": {   "single_paired|in1": {    "id": "06e5c12644eb3d77",    "src": "hda",    "uuid": "d188080c-bfad-4375-aa44-84f9973628c8"   }  },  "job_messages": null,  "job_metrics": [],  "job_runner_name": null,  "job_stderr": null,  "job_stdout": null,  "model_class": "Job",  "output_collections": {},  "outputs": {   "out1": {    "id": "36fb988cdba5ca71",    "src": "hda",    "uuid": "b5fa9236-471f-4ae7-8691-217b9a534eef"   },   "report_html": {    "id": "f0f82aacd6f51df6",    "src": "hda",    "uuid": "5b5f550e-cd21-41e8-9b0c-97cadd35d7e1"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "filter_options": "{\"length_filtering_options\": {\"disable_length_filtering\": false, \"length_limit\": \"\", \"length_required\": \"\"}, \"low_complexity_filter\": {\"complexity_threshold\": \"\", \"enable_low_complexity_filter\": false}, \"quality_filtering_options\": {\"disable_quality_filtering\": false, \"n_base_limit\": \"\", \"qualified_quality_phred\": \"\", \"unqualified_percent_limit\": \"\"}}",   "output_options": "{\"report_html\": true, \"report_json\": false}",   "overrepresented_sequence_analysis": "{\"overrepresentation_analysis\": false, \"overrepresentation_sampling\": \"\"}",   "read_mod_options": "{\"base_correction_options\": {\"correction\": false}, \"cutting_by_quality_options\": {\"cut_by_quality3\": false, \"cut_by_quality5\": false, \"cut_mean_quality\": \"\", \"cut_window_size\": \"\"}, \"polyg_tail_trimming\": {\"__current_case__\": 1, \"poly_g_min_len\": \"\", \"trimming_select\": \"\"}, \"polyx_tail_trimming\": {\"__current_case__\": 1, \"polyx_trimming_select\": \"\"}, \"umi_processing\": {\"umi\": true, \"umi_len\": \"8\", \"umi_loc\": \"read1\", \"umi_prefix\": null}}",   "single_paired": "{\"__current_case__\": 0, \"adapter_trimming_options\": {\"adapter_sequence1\": null, \"disable_adapter_trimming\": false}, \"global_trimming_options\": {\"trim_front1\": \"\", \"trim_tail1\": \"\"}, \"in1\": {\"values\": [{\"id\": 11, \"src\": \"hda\"}]}, \"single_paired_selector\": \"single\"}"  },  "state": "error",  "stderr": "Detecting adapter sequence for read1...\nNo adapter detected for read1\n\nRead1 before filtering:\ntotal reads: 2\ntotal bases: 302\nQ20 bases: 262(86.755%)\nQ30 bases: 241(79.8013%)\n\nRead1 after filtering:\ntotal reads: 2\ntotal bases: 286\nQ20 bases: 246(86.014%)\nQ30 bases: 226(79.021%)\n\nFiltering result:\nreads passed filter: 2\nreads failed due to low quality: 0\nreads failed due to too many N: 0\nreads failed due to too short: 0\nreads with adapter trimmed: 0\nbases trimmed due to adapters: 0\n\nDuplication rate (may be overestimated since this is SE data): 0%\n\nJSON report: fastp.json\nHTML report: fastp.html\n\nfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger -U --umi_loc read1 --umi_len 8 \nfastp v0.24.0, time used: 0 seconds\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.24.0+galaxy3",  "tool_stderr": "Detecting adapter sequence for read1...\nNo adapter detected for read1\n\nRead1 before filtering:\ntotal reads: 2\ntotal bases: 302\nQ20 bases: 262(86.755%)\nQ30 bases: 241(79.8013%)\n\nRead1 after filtering:\ntotal reads: 2\ntotal bases: 286\nQ20 bases: 246(86.014%)\nQ30 bases: 226(79.021%)\n\nFiltering result:\nreads passed filter: 2\nreads failed due to low quality: 0\nreads failed due to too many N: 0\nreads failed due to too short: 0\nreads with adapter trimmed: 0\nbases trimmed due to adapters: 0\n\nDuplication rate (may be overestimated since this is SE data): 0%\n\nJSON report: fastp.json\nHTML report: fastp.html\n\nfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger -U --umi_loc read1 --umi_len 8 \nfastp v0.24.0, time used: 0 seconds\n",  "tool_stdout": "",  "update_time": "2025-06-20T01:02:23.572945",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.24.0+galaxy3, exit_code: None, stderr: Detecting adapter sequence for read1...\nNo adapter detected for read1\n\nRead1 before filtering:\ntotal reads: 2\ntotal bases: 302\nQ20 bases: 262(86.755%)\nQ30 bases: 241(79.8013%)\n\nRead1 after filtering:\ntotal reads: 2\ntotal bases: 286\nQ20 bases: 246(86.014%)\nQ30 bases: 226(79.021%)\n\nFiltering result:\nreads passed filter: 2\nreads failed due to low quality: 0\nreads failed due to too many N: 0\nreads failed due to too short: 0\nreads with adapter trimmed: 0\nbases trimmed due to adapters: 0\n\nDuplication rate (may be overestimated since this is SE data): 0%\n\nJSON report: fastp.json\nHTML report: fastp.html\n\nfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger -U --umi_loc read1 --umi_len 8 \nfastp v0.24.0, time used: 0 seconds\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.24.0+galaxy3, exit_code: None, stderr: Detecting adapter sequence for read1...\nNo adapter detected for read1\n\nRead1 before filtering:\ntotal reads: 2\ntotal bases: 302\nQ20 bases: 262(86.755%)\nQ30 bases: 241(79.8013%)\n\nRead1 after filtering:\ntotal reads: 2\ntotal bases: 286\nQ20 bases: 246(86.014%)\nQ30 bases: 226(79.021%)\n\nFiltering result:\nreads passed filter: 2\nreads failed due to low quality: 0\nreads failed due to too many N: 0\nreads failed due to too short: 0\nreads with adapter trimmed: 0\nbases trimmed due to adapters: 0\n\nDuplication rate (may be overestimated since this is SE data): 0%\n\nJSON report: fastp.json\nHTML report: fastp.html\n\nfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger -U --umi_loc read1 --umi_len 8 \nfastp v0.24.0, time used: 0 seconds\n." ], "status": "failure", "test_index": 3, "time_seconds": 26.559338092803955, "tool_id": "fastp", "tool_version": "0.24.0+galaxy3"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Detecting adapter sequence for read1...No adapter detected for read1Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 2total bases: 286Q20 bases: 246(86.014%)Q30 bases: 226(79.021%)Filtering result:reads passed filter: 2reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 0bases trimmed due to adapters: 0Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger -U --umi_loc read1 --umi_len 8 fastp v0.24.0, time used: 0 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #5)" name="4" time="34.19809889793396">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Detecting adapter sequence for read1...No adapter detected for read1Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 2total bases: 278Q20 bases: 239(85.9712%)Q30 bases: 220(79.1367%)Filtering result:reads passed filter: 2reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 0bases trimmed due to adapters: 0Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger -U --umi_loc read1 --umi_len 12 fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #6)" name="5" time="34.44551944732666">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 99total bases: 24849Q20 bases: 23268(93.6376%)Q30 bases: 22391(90.1083%)Read2 before filtering:total reads: 99total bases: 24849Q20 bases: 22676(91.2552%)Q30 bases: 21532(86.6514%)Read1 after filtering:total reads: 97total bases: 22942Q20 bases: 21680(94.4992%)Q30 bases: 20981(91.4524%)Read2 after filtering:total reads: 97total bases: 22942Q20 bases: 21535(93.8671%)Q30 bases: 20672(90.1055%)Filtering result:reads passed filter: 194reads failed due to low quality: 2reads failed due to too many N: 2reads failed due to too short: 0reads with adapter trimmed: 68bases trimmed due to adapters: 2824Duplication rate: 0%Insert size peak (evaluated by paired-end reads): 308JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for bwa-mem-fastq1_fq.fastqsanger -i bwa-mem-fastq1_fq.fastqsanger -I bwa-mem-fastq2_fq_R2.fastqsanger -o first.fastqsanger -O second.fastqsanger fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #7)" name="6" time="35.55115532875061">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 99total bases: 24849Q20 bases: 23268(93.6376%)Q30 bases: 22391(90.1083%)Read2 before filtering:total reads: 99total bases: 24849Q20 bases: 22676(91.2552%)Q30 bases: 21532(86.6514%)Read1 after filtering:total reads: 97total bases: 22166Q20 bases: 20935(94.4464%)Q30 bases: 20255(91.3787%)Read2 after filtering:total reads: 97total bases: 22646Q20 bases: 21284(93.9857%)Q30 bases: 20447(90.2897%)Filtering result:reads passed filter: 194reads failed due to low quality: 2reads failed due to too many N: 2reads failed due to too short: 0reads with adapter trimmed: 76bases trimmed due to adapters: 3128Duplication rate: 0%Insert size peak (evaluated by paired-end reads): 243JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for bwa-mem-fastq1_fq.fastqsanger -i bwa-mem-fastq1_fq.fastqsanger -I bwa-mem-fastq2_fq_R2.fastqsanger -o first.fastqsanger -O second.fastqsanger -U --umi_loc read1 --umi_len 8 fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #8)" name="7" time="35.420756578445435">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Read1 before filtering:total reads: 99total bases: 24849Q20 bases: 23268(93.6376%)Q30 bases: 22391(90.1083%)Read2 before filtering:total reads: 99total bases: 24849Q20 bases: 22676(91.2552%)Q30 bases: 21532(86.6514%)Read1 after filtering:total reads: 97total bases: 22678Q20 bases: 21439(94.5366%)Q30 bases: 20756(91.5248%)Read2 after filtering:total reads: 97total bases: 22166Q20 bases: 20766(93.684%)Q30 bases: 19910(89.8223%)Filtering result:reads passed filter: 194reads failed due to low quality: 2reads failed due to too many N: 2reads failed due to too short: 0reads with adapter trimmed: 68bases trimmed due to adapters: 3096Duplication rate: 0%Insert size peak (evaluated by paired-end reads): 300JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for bwa-mem-fastq1_fq.fastqsanger -i bwa-mem-fastq1_fq.fastqsanger -I bwa-mem-fastq2_fq_R2.fastqsanger -o first.fastqsanger -O second.fastqsanger -U --umi_loc read2 --umi_len 8 fastp v0.24.0, time used: 0 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #9)" name="8" time="32.8869948387146">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Detecting adapter sequence for read1...No adapter detected for read1Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Filtering result:reads passed filter: 2reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 0bases trimmed due to adapters: 0Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq.fastqsanger -i R1_fq.fastqsanger -o first.fastqsanger fastp v0.24.0, time used: 0 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="fastp (Test #10)" name="9" time="35.99350833892822">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Detecting adapter sequence for read1...No adapter detected for read1Read1 before filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Read1 after filtering:total reads: 2total bases: 302Q20 bases: 262(86.755%)Q30 bases: 241(79.8013%)Filtering result:reads passed filter: 2reads failed due to low quality: 0reads failed due to too many N: 0reads failed due to too short: 0reads with adapter trimmed: 0bases trimmed due to adapters: 0Duplication rate (may be overestimated since this is SE data): 0%JSON report: fastp.jsonHTML report: fastp.htmlfastp --thread 4 --report_title fastp report for R1_fq_gz.fastqsanger.gz -i R1_fq_gz.fastqsanger.gz -o first.fastqsanger.gz --poly_g_min_len 10 -g fastp v0.24.0, time used: 1 seconds
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="ivar_variants (Test #1)" name="0" time="34.17969012260437">
        
            
                <error type="error" message="Tool exit code: None"><![CDATA[
                    { "inputs": {  "input_bam": {   "id": "67f2e8f256856609",   "src": "hda"  },  "ref": {   "id": "f30517a0c275c175",   "src": "hda"  } }, "job": {  "command_line": "ln -s \u0027/galaxy/server/database/objects/0/0/f/dataset_00f04777-6f97-4d52-8e35-c47c0e3d31dc.dat\u0027 ref.fa \u0026\u0026 ln -s \u0027/galaxy/server/database/objects/c/8/0/dataset_c8082e33-004d-4584-b20b-a399f4117a0a.dat\u0027 sorted.bam \u0026\u0026 samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam | ivar variants -p variants -q 20 -t 0.03",  "command_version": null,  "copied_from_job_id": null,  "create_time": "2025-06-20T01:24:11.783641",  "dependencies": [],  "exit_code": null,  "external_id": "gxy-2nq48",  "galaxy_version": "24.1",  "handler": null,  "history_id": "62db28f22a512698",  "id": "7233b5984984c0d8",  "inputs": {   "input_bam": {    "id": "67f2e8f256856609",    "src": "hda",    "uuid": "c8082e33-004d-4584-b20b-a399f4117a0a"   },   "ref": {    "id": "f30517a0c275c175",    "src": "hda",    "uuid": "00f04777-6f97-4d52-8e35-c47c0e3d31dc"   }  },  "job_messages": null,  "job_metrics": [],  "job_runner_name": null,  "job_stderr": null,  "job_stdout": null,  "model_class": "Job",  "output_collections": {},  "outputs": {   "output_variants_tabular": {    "id": "8e0ae5e12e14301f",    "src": "hda",    "uuid": "e8e42fa6-5020-4ddf-9a30-2544875d39cb"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "min_freq": "\"0.03\"",   "min_qual": "\"20\"",   "output_format": "{\"__current_case__\": 2, \"choice\": \"tabular\", \"gtf\": null}"  },  "state": "error",  "stderr": "[mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n",  "stdout": "A GFF file containing the open reading frames (ORFs) has not been provided. Amino acid translation will not be done.\nA reference sequence has not been supplied. Amino acid translation will not be done.\n",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/ivar_variants/ivar_variants/1.4.3+galaxy1",  "tool_stderr": "[mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n",  "tool_stdout": "A GFF file containing the open reading frames (ORFs) has not been provided. Amino acid translation will not be done.\nA reference sequence has not been supplied. Amino acid translation will not be done.\n",  "update_time": "2025-06-20T01:24:25.355011",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/ivar_variants/ivar_variants/1.4.3+galaxy1, exit_code: None, stderr: [mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/ivar_variants/ivar_variants/1.4.3+galaxy1, exit_code: None, stderr: [mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n." ], "status": "failure", "test_index": 0, "time_seconds": 34.17969012260437, "tool_id": "ivar_variants", "tool_version": "1.4.3+galaxy1"}
                ]]></error>
            
            <system-out><![CDATA[
            A GFF file containing the open reading frames (ORFs) has not been provided. Amino acid translation will not be done.A reference sequence has not been supplied. Amino acid translation will not be done.
            ]]></system-out>
            <system-err><![CDATA[
            [mpileup] 1 samples in 1 input files[mpileup] Max depth set to maximum value (2147483647)
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="ivar_variants (Test #2)" name="1" time="35.24436807632446">
        
            
            <system-out><![CDATA[
            A GFF file containing the open reading frames (ORFs) has not been provided. Amino acid translation will not be done.A reference sequence has not been supplied. Amino acid translation will not be done.sampleDELINSSNPvariants0115
            ]]></system-out>
            <system-err><![CDATA[
            [mpileup] 1 samples in 1 input files[mpileup] Max depth set to maximum value (2147483647)
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="ivar_variants (Test #3)" name="2" time="36.40384840965271">
        
            
            <system-out><![CDATA[
            A GFF file containing the open reading frames (ORFs) has not been provided. Amino acid translation will not be done.A reference sequence has not been supplied. Amino acid translation will not be done.sampleDELINSSNPvariants0010
            ]]></system-out>
            <system-err><![CDATA[
            [mpileup] 1 samples in 1 input files[mpileup] Max depth set to maximum value (2147483647)
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="ivar_variants (Test #4)" name="3" time="43.343544721603394">
        
            
                <error type="error" message="Tool exit code: 139"><![CDATA[
                    { "inputs": {  "input_bam": {   "id": "937a81d106044f5b",   "src": "hda"  },  "output_format|gtf": {   "id": "67692b092fdf5692",   "src": "hda"  },  "ref": {   "id": "b68706b217082c6a",   "src": "hda"  } }, "job": {  "command_line": "ln -s \u0027/galaxy/server/database/objects/5/e/8/dataset_5e8462e0-b297-4114-80bc-e55cccbfa262.dat\u0027 ref.fa \u0026\u0026 ln -s \u0027/galaxy/server/database/objects/c/0/d/dataset_c0d16db1-934f-47fd-bb7f-085fd98b5348.dat\u0027 sorted.bam \u0026\u0026 samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam | ivar variants -p variants -q 20 -t 0.03 -r ref.fa -g \u0027/galaxy/server/database/objects/5/d/7/dataset_5d75605e-27e1-4678-b9af-7f1d0bfde7c6.dat\u0027",  "command_version": "iVar version 1.4.3",  "copied_from_job_id": null,  "create_time": "2025-06-20T01:26:07.614282",  "dependencies": [],  "exit_code": 139,  "external_id": "gxy-m4jcn",  "galaxy_version": "24.1",  "handler": null,  "history_id": "62db28f22a512698",  "id": "6d83b45ce0841cae",  "inputs": {   "input_bam": {    "id": "937a81d106044f5b",    "src": "hda",    "uuid": "c0d16db1-934f-47fd-bb7f-085fd98b5348"   },   "output_format|gtf": {    "id": "67692b092fdf5692",    "src": "hda",    "uuid": "5d75605e-27e1-4678-b9af-7f1d0bfde7c6"   },   "ref": {    "id": "b68706b217082c6a",    "src": "hda",    "uuid": "5e8462e0-b297-4114-80bc-e55cccbfa262"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 139 ()",    "error_level": 3,    "exit_code": 139,    "type": "exit_code"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "0E-7",    "title": "Job Runtime (Wall Clock)",    "value": "0 seconds"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1750382769.0000000",    "title": "Job End Time",    "value": "2025-06-20 01:26:09"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1750382769.0000000",    "title": "Job Start Time",    "value": "2025-06-20 01:26:09"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "32641.0000000",    "title": "Memory Allocated (MB)",    "value": "32641"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   }  ],  "job_runner_name": null,  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output_variants_tabular": {    "id": "db4dbfc4a9cdce83",    "src": "hda",    "uuid": "c092d199-7549-4a75-b497-d396bb09f734"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "min_freq": "\"0.03\"",   "min_qual": "\"20\"",   "output_format": "{\"__current_case__\": 2, \"choice\": \"tabular\", \"gtf\": {\"values\": [{\"id\": 143, \"src\": \"hda\"}]}}"  },  "state": "error",  "stderr": "[mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n/galaxy/server/database/jobs_directory/000/115/tool_script.sh: line 24:    27 Broken pipe             samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam\n        28 Segmentation fault      (core dumped) | ivar variants -p variants -q 20 -t 0.03 -r ref.fa -g \u0027/galaxy/server/database/objects/5/d/7/dataset_5d75605e-27e1-4678-b9af-7f1d0bfde7c6.dat\u0027\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/ivar_variants/ivar_variants/1.4.3+galaxy1",  "tool_stderr": "[mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n/galaxy/server/database/jobs_directory/000/115/tool_script.sh: line 24:    27 Broken pipe             samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam\n        28 Segmentation fault      (core dumped) | ivar variants -p variants -q 20 -t 0.03 -r ref.fa -g \u0027/galaxy/server/database/objects/5/d/7/dataset_5d75605e-27e1-4678-b9af-7f1d0bfde7c6.dat\u0027\n",  "tool_stdout": "",  "update_time": "2025-06-20T01:26:20.676275",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/ivar_variants/ivar_variants/1.4.3+galaxy1, exit_code: 139, stderr: [mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n/galaxy/server/database/jobs_directory/000/115/tool_script.sh: line 24:    27 Broken pipe             samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam\n        28 Segmentation fault      (core dumped) | ivar variants -p variants -q 20 -t 0.03 -r ref.fa -g \u0027/galaxy/server/database/objects/5/d/7/dataset_5d75605e-27e1-4678-b9af-7f1d0bfde7c6.dat\u0027\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/ivar_variants/ivar_variants/1.4.3+galaxy1, exit_code: 139, stderr: [mpileup] 1 samples in 1 input files\n[mpileup] Max depth set to maximum value (2147483647)\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence\n/galaxy/server/database/jobs_directory/000/115/tool_script.sh: line 24:    27 Broken pipe             samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam\n        28 Segmentation fault      (core dumped) | ivar variants -p variants -q 20 -t 0.03 -r ref.fa -g \u0027/galaxy/server/database/objects/5/d/7/dataset_5d75605e-27e1-4678-b9af-7f1d0bfde7c6.dat\u0027\n." ], "status": "failure", "test_index": 3, "time_seconds": 43.343544721603394, "tool_id": "ivar_variants", "tool_version": "1.4.3+galaxy1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [mpileup] 1 samples in 1 input files[mpileup] Max depth set to maximum value (2147483647)[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence[W::fai_get_val] Reference PRV not found in FASTA file, returning empty sequence/galaxy/server/database/jobs_directory/000/115/tool_script.sh: line 24:    27 Broken pipe             samtools mpileup -A -d 0 --reference ref.fa -B -Q 0 sorted.bam        28 Segmentation fault      (core dumped) | ivar variants -p variants -q 20 -t 0.03 -r ref.fa -g '/galaxy/server/database/objects/5/d/7/dataset_5d75605e-27e1-4678-b9af-7f1d0bfde7c6.dat'
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="ivar_variants (Test #5)" name="4" time="39.87730669975281">
        
            
            <system-out><![CDATA[
            sampleDELINSSNPvariants0010
            ]]></system-out>
            <system-err><![CDATA[
            [mpileup] 1 samples in 1 input files[mpileup] Max depth set to maximum value (2147483647)
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="lofreq_viterbi (Test #1)" name="0" time="45.19058084487915">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [bam_sort_core] merging from 0 files and 4 in-memory blocks...
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="lofreq_viterbi (Test #2)" name="1" time="33.28567147254944">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [bam_sort_core] merging from 0 files and 4 in-memory blocks...
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="poretools_events (Test #1)" name="0" time="77.36865639686584">
        
            
                <error type="error" message="Tool exit code: 1"><![CDATA[
                    { "inputs": {  "input": {   "id": "b642f56f311753ba",   "src": "hda"  },  "precalled": false }, "job": {  "command_line": "poretools events  \u0027/galaxy/server/database/objects/a/f/8/dataset_af8b236f-a777-4c70-b6b4-4141e8fa61ee.dat\u0027 \u003e \u0027/galaxy/server/database/objects/1/5/1/dataset_151aaf38-cc10-4491-a4f9-0915392481df.dat\u0027",  "command_version": "",  "copied_from_job_id": null,  "create_time": "2025-06-20T01:19:31.235736",  "dependencies": [],  "exit_code": 1,  "external_id": "gxy-4dqfw",  "galaxy_version": "24.1",  "handler": null,  "history_id": "62db28f22a512698",  "id": "63adafe6886f9a9c",  "inputs": {   "input": {    "id": "b642f56f311753ba",    "src": "hda",    "uuid": "af8b236f-a777-4c70-b6b4-4141e8fa61ee"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 1 ()",    "error_level": 3,    "exit_code": 1,    "type": "exit_code"   },   {    "code_desc": "",    "desc": "Fatal error: ",    "error_level": 3,    "match": "Error:",    "stream": "stderr",    "type": "regex"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Job Runtime (Wall Clock)",    "value": "1 second"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1750382410.0000000",    "title": "Job End Time",    "value": "2025-06-20 01:20:10"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1750382409.0000000",    "title": "Job Start Time",    "value": "2025-06-20 01:20:09"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "12240.0000000",    "title": "Memory Allocated (MB)",    "value": "12240"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   }  ],  "job_runner_name": null,  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "78ff5cc2e9b0195e",    "src": "hda",    "uuid": "151aaf38-cc10-4491-a4f9-0915392481df"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "precalled": "false"  },  "state": "error",  "stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1",  "tool_stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "tool_stdout": "",  "update_time": "2025-06-20T01:20:28.150585",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n." ], "status": "failure", "test_index": 0, "time_seconds": 77.36865639686584, "tool_id": "poretools_events", "tool_version": "0.6.1a1.1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Traceback (most recent call last):  File "/usr/local/bin/poretools", line 11, in <module>    load_entry_point('poretools==0.6.0', 'console_scripts', 'poretools')()  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 543, in main    args.func(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 55, in run_subtool    submodule.run(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/events.py", line 17, in run    for fast5 in Fast5File.Fast5FileSet(args.files):  File "/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py", line 110, in next    shutil.rmtree(PORETOOLS_TMPDIR)  File "/usr/local/lib/python2.7/shutil.py", line 261, in rmtree    rmtree(fullname, ignore_errors, onerror)  File "/usr/local/lib/python2.7/shutil.py", line 270, in rmtree    onerror(os.rmdir, path, sys.exc_info())  File "/usr/local/lib/python2.7/shutil.py", line 268, in rmtree    os.rmdir(path)OSError: [Errno 39] Directory not empty: '.poretools_tmp/test-data'
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="poretools_events (Test #2)" name="1" time="32.84616494178772">
        
            
                <error type="error" message="Tool exit code: 1"><![CDATA[
                    { "inputs": {  "input": {   "id": "f276e5689b2483b9",   "src": "hda"  },  "precalled": true }, "job": {  "command_line": "poretools events --pre-basecalled \u0027/galaxy/server/database/objects/9/e/f/dataset_9efff169-0271-4169-96c6-96272e805345.dat\u0027 \u003e \u0027/galaxy/server/database/objects/2/8/a/dataset_28a09215-e8f7-4d9c-b336-41562c771f93.dat\u0027",  "command_version": "",  "copied_from_job_id": null,  "create_time": "2025-06-20T01:20:48.045949",  "dependencies": [],  "exit_code": 1,  "external_id": "gxy-f46zx",  "galaxy_version": "24.1",  "handler": null,  "history_id": "62db28f22a512698",  "id": "73f36fdd772661a1",  "inputs": {   "input": {    "id": "f276e5689b2483b9",    "src": "hda",    "uuid": "9efff169-0271-4169-96c6-96272e805345"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 1 ()",    "error_level": 3,    "exit_code": 1,    "type": "exit_code"   },   {    "code_desc": "",    "desc": "Fatal error: ",    "error_level": 3,    "match": "Error:",    "stream": "stderr",    "type": "regex"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Job Runtime (Wall Clock)",    "value": "1 second"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1750382450.0000000",    "title": "Job End Time",    "value": "2025-06-20 01:20:50"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1750382449.0000000",    "title": "Job Start Time",    "value": "2025-06-20 01:20:49"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "12240.0000000",    "title": "Memory Allocated (MB)",    "value": "12240"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   }  ],  "job_runner_name": null,  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "a2a080c3c8640aad",    "src": "hda",    "uuid": "28a09215-e8f7-4d9c-b336-41562c771f93"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "precalled": "true"  },  "state": "error",  "stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1",  "tool_stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "tool_stdout": "",  "update_time": "2025-06-20T01:21:01.087390",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n." ], "status": "failure", "test_index": 1, "time_seconds": 32.84616494178772, "tool_id": "poretools_events", "tool_version": "0.6.1a1.1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Traceback (most recent call last):  File "/usr/local/bin/poretools", line 11, in <module>    load_entry_point('poretools==0.6.0', 'console_scripts', 'poretools')()  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 543, in main    args.func(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 55, in run_subtool    submodule.run(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/events.py", line 13, in run    for fast5 in Fast5File.Fast5FileSet(args.files):  File "/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py", line 110, in next    shutil.rmtree(PORETOOLS_TMPDIR)  File "/usr/local/lib/python2.7/shutil.py", line 261, in rmtree    rmtree(fullname, ignore_errors, onerror)  File "/usr/local/lib/python2.7/shutil.py", line 270, in rmtree    onerror(os.rmdir, path, sys.exc_info())  File "/usr/local/lib/python2.7/shutil.py", line 268, in rmtree    os.rmdir(path)OSError: [Errno 39] Directory not empty: '.poretools_tmp/test-data'
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_merge (Test #1)" name="0" time="44.18920350074768">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_merge (Test #2)" name="1" time="37.94957637786865">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [bam_translate] PG tag "colt" on read "r001" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_merge (Test #3)" name="2" time="32.290634632110596">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_merge (Test #4)" name="3" time="25.667375087738037">
        
            
                <error type="error" message="Tool exit code: None"><![CDATA[
                    { "inputs": {  "bamfiles": [   {    "id": "9e4219559c1d8b93",    "src": "hda"   },   {    "id": "894afa10def37031",    "src": "hda"   }  ],  "idpg": true,  "idrg": true }, "job": {  "command_line": "addthreads=${GALAXY_SLOTS:-1} \u0026\u0026 (( addthreads-- )) \u0026\u0026   ln -s \u0027/galaxy/server/database/objects/6/c/6/dataset_6c6877eb-a4a2-4783-8605-9f5531705697.dat\u0027 0.sam \u0026\u0026  ln -s \u0027/galaxy/server/database/objects/3/5/0/dataset_350c4752-6a30-45ee-a8f7-73bd85e741fe.dat\u0027 1.sam \u0026\u0026   samtools merge -@ $addthreads -s 1 -f -c -p /galaxy/server/database/objects/9/9/7/dataset_997bdedb-51c2-40b8-af18-84fe3a7efcf9.dat 0.sam 1.sam",  "command_version": null,  "copied_from_job_id": null,  "create_time": "2025-06-20T01:11:04.963531",  "dependencies": [],  "exit_code": null,  "external_id": "gxy-t5wbj",  "galaxy_version": "24.1",  "handler": null,  "history_id": "62db28f22a512698",  "id": "764fdd315c2be4f5",  "inputs": {   "bamfiles": {    "id": "9e4219559c1d8b93",    "src": "hda",    "uuid": "6c6877eb-a4a2-4783-8605-9f5531705697"   },   "bamfiles1": {    "id": "9e4219559c1d8b93",    "src": "hda",    "uuid": "6c6877eb-a4a2-4783-8605-9f5531705697"   },   "bamfiles2": {    "id": "894afa10def37031",    "src": "hda",    "uuid": "350c4752-6a30-45ee-a8f7-73bd85e741fe"   }  },  "job_messages": null,  "job_metrics": [],  "job_runner_name": null,  "job_stderr": null,  "job_stdout": null,  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "3d87be767ca38435",    "src": "hda",    "uuid": "997bdedb-51c2-40b8-af18-84fe3a7efcf9"   }  },  "params": {   "__input_ext": "\"bam\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "headerbam": null,   "idpg": "true",   "idrg": "true",   "region": null,   "seed": "\"1\""  },  "state": "error",  "stderr": "[bam_translate] PG tag \"colt\" on read \"r001\" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/samtools_merge/samtools_merge/1.9",  "tool_stderr": "[bam_translate] PG tag \"colt\" on read \"r001\" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.\n",  "tool_stdout": "",  "update_time": "2025-06-20T01:11:10.503837",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/samtools_merge/samtools_merge/1.9, exit_code: None, stderr: [bam_translate] PG tag \"colt\" on read \"r001\" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/samtools_merge/samtools_merge/1.9, exit_code: None, stderr: [bam_translate] PG tag \"colt\" on read \"r001\" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.\n." ], "status": "failure", "test_index": 3, "time_seconds": 25.667375087738037, "tool_id": "samtools_merge", "tool_version": "1.9"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [bam_translate] PG tag "colt" on read "r001" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_merge (Test #5)" name="4" time="33.62854290008545">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            [bam_translate] PG tag "colt" on read "r001" encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="vcfannotategenotypes (Test #1)" name="0" time="42.1745707988739">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="vcfsort (Test #1)" name="0" time="33.84678602218628">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="volcanoplot (Test #1)" name="0" time="11.09431004524231">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1434, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 747, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 839, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1434, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 747, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 839, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 0, "time_seconds": 11.09431004524231, "tool_id": "volcanoplot", "tool_version": "0.0.7"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="volcanoplot (Test #2)" name="1" time="11.891064643859863">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1434, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 747, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 839, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1434, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 747, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 839, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 1, "time_seconds": 11.891064643859863, "tool_id": "volcanoplot", "tool_version": "0.0.7"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="volcanoplot (Test #3)" name="2" time="59.59778022766113">
        
            
            <system-out><![CDATA[
            null device           1 R version 4.0.5 (2021-03-31)Platform: x86_64-conda-linux-gnu (64-bit)Running under: Debian GNU/Linux 10 (buster)Matrix products: defaultBLAS/LAPACK: /usr/local/lib/libopenblasp-r0.3.15.solocale: [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8    [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   attached base packages:[1] stats     graphics  grDevices utils     datasets  methods   base     other attached packages:[1] ggrepel_0.9.1 ggplot2_3.3.3 dplyr_1.0.6  loaded via a namespace (and not attached): [1] Rcpp_1.0.6       magrittr_2.0.1   tidyselect_1.1.1 munsell_0.5.0    [5] colorspace_2.0-1 R6_2.5.0         rlang_0.4.11     fansi_0.5.0      [9] grid_4.0.5       gtable_0.3.0     utf8_1.2.1       withr_2.4.2     [13] ellipsis_0.3.2   digest_0.6.27    tibble_3.1.2     lifecycle_1.0.0 [17] crayon_1.4.1     purrr_0.3.4      farver_2.1.0     vctrs_0.3.8     [21] glue_1.4.2       labeling_0.4.2   compiler_4.0.5   pillar_1.6.1    [25] generics_0.1.0   scales_1.1.1     pkgconfig_2.0.3 
            ]]></system-out>
            <system-err><![CDATA[
            Warning message:In Sys.setlocale("LC_MESSAGES", "en_US.UTF-8") :  OS reports request to set locale to "en_US.UTF-8" cannot be honored
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="volcanoplot (Test #4)" name="3" time="36.930262327194214">
        
            
            <system-out><![CDATA[
            [1] "Header row detected"null device           1 R version 4.0.5 (2021-03-31)Platform: x86_64-conda-linux-gnu (64-bit)Running under: Debian GNU/Linux 10 (buster)Matrix products: defaultBLAS/LAPACK: /usr/local/lib/libopenblasp-r0.3.15.solocale: [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8    [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   attached base packages:[1] stats     graphics  grDevices utils     datasets  methods   base     other attached packages:[1] ggrepel_0.9.1 ggplot2_3.3.3 dplyr_1.0.6  loaded via a namespace (and not attached): [1] Rcpp_1.0.6       magrittr_2.0.1   tidyselect_1.1.1 munsell_0.5.0    [5] colorspace_2.0-1 R6_2.5.0         rlang_0.4.11     fansi_0.5.0      [9] grid_4.0.5       gtable_0.3.0     utf8_1.2.1       withr_2.4.2     [13] ellipsis_0.3.2   digest_0.6.27    tibble_3.1.2     lifecycle_1.0.0 [17] crayon_1.4.1     purrr_0.3.4      farver_2.1.0     vctrs_0.3.8     [21] glue_1.4.2       labeling_0.4.2   compiler_4.0.5   pillar_1.6.1    [25] generics_0.1.0   scales_1.1.1     pkgconfig_2.0.3 
            ]]></system-out>
            <system-err><![CDATA[
            Warning message:In Sys.setlocale("LC_MESSAGES", "en_US.UTF-8") :  OS reports request to set locale to "en_US.UTF-8" cannot be honored
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="volcanoplot (Test #5)" name="4" time="36.73381972312927">
        
            
            <system-out><![CDATA[
            [1] "Header row detected"null device           1 R version 4.0.5 (2021-03-31)Platform: x86_64-conda-linux-gnu (64-bit)Running under: Debian GNU/Linux 10 (buster)Matrix products: defaultBLAS/LAPACK: /usr/local/lib/libopenblasp-r0.3.15.solocale: [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8    [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   attached base packages:[1] stats     graphics  grDevices utils     datasets  methods   base     other attached packages:[1] ggrepel_0.9.1 ggplot2_3.3.3 dplyr_1.0.6  loaded via a namespace (and not attached): [1] Rcpp_1.0.6       magrittr_2.0.1   tidyselect_1.1.1 munsell_0.5.0    [5] colorspace_2.0-1 R6_2.5.0         rlang_0.4.11     fansi_0.5.0      [9] grid_4.0.5       gtable_0.3.0     utf8_1.2.1       withr_2.4.2     [13] ellipsis_0.3.2   digest_0.6.27    tibble_3.1.2     lifecycle_1.0.0 [17] crayon_1.4.1     purrr_0.3.4      farver_2.1.0     vctrs_0.3.8     [21] glue_1.4.2       labeling_0.4.2   compiler_4.0.5   pillar_1.6.1    [25] generics_0.1.0   scales_1.1.1     pkgconfig_2.0.3 
            ]]></system-out>
            <system-err><![CDATA[
            Warning message:In Sys.setlocale("LC_MESSAGES", "en_US.UTF-8") :  OS reports request to set locale to "en_US.UTF-8" cannot be honored
            ]]></system-err>
        
    </testcase>
    
</testsuite>